experimental/oak_functions_with_envoy/client/client.yaml

```diff
@@ -0,0 +1,47 @@
+static_resources:
+  listeners:
+    # Accepts TCP streams and tunnels them over HTTP streams.
+    # https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/http/upgrades#tunneling-tcp-over-http
+    - name: client_listener
+      address:
+        # Client proxy address to listen on.
+        socket_address:
+          protocol: TCP
+          address: 0.0.0.0
+          port_value: 8000
+      filter_chains:
+        - filters:
+            - name: tcp
+              typed_config:
+                '@type': type.googleapis.com/envoy.extensions.filters.network.tcp_proxy.v3.TcpProxy
+                stat_prefix: tcp_stats
+                cluster: client_cluster
+                tunneling_config:
+                  hostname: oak-functions-with-envoy-example-62sa4xcfia-nw.a.run.app
+                  # Send TCP segments in POST HTTP requests.
+                  use_post: true
+
+  clusters:
+    # Connects to the remote server proxy via Cloud Run, which terminated the TLS connection.
+    - name: client_cluster
+      connect_timeout: 5s
+      type: LOGICAL_DNS
+      http2_protocol_options: {}
+      load_assignment:
+        cluster_name: client_cluster
+        endpoints:
+          - lb_endpoints:
+              - endpoint:
+                  address:
+                    # Remote server proxy address to connect to.
+                    socket_address:
+                      address: oak-functions-with-envoy-example-62sa4xcfia-nw.a.run.app
```

Comment:
I am not sure why this address is repeated twice, are there two different kinds of connections going on at the same time?